packages/blockchain-wallet-v4-frontend/src/data/modules/sfox/sagas.js

```diff
@@ -100,14 +105,49 @@ export default ({ coreSagas }) => {
   }
 
   const submitSellQuote = function * (action) {
+    const q = action.payload
     try {
-      console.log('submitting sell quote:', action.payload)
-      yield call(coreSagas.data.sfox.handleSellTrade, action.payload)
-      let state = yield select()
-      console.log('state', state)
+      console.log('submitting sell quote:', q)
+
+      // we will need the trade object for the 'to' address and id
+      const trade = yield call(coreSagas.data.sfox.handleSellTrade, action.payload)
+      console.log('trade response', trade)
+
+      let p = yield select(sendBtcSelectors.getPayment)
+      let payment = yield coreSagas.payment.btc.create({ payment: p.getOrElse({}), network: settings.NETWORK_BITCOIN })
+
+      // assign payment amount from quote
+      payment = yield payment.amount(parseInt(trade.sendAmount))
+
+      // use priority fee
+      payment = yield payment.fee('priority')
+
+      // assign payment to from trade
+      payment = yield payment.to('153tQjKYMuxRhymRDvoHqcKez94anDGkGF') // TODO this should be "trade.receiveAddress"
+
+      // assign description with id from trade
+      // let trade = { id: 'fakeId12345' }
```

Comment:
unnecessary comment?

Reply:
For sure, this will get cleaned up before merging into main sell branch.